app/controllers/report_controller/usage.rb

```diff
@@ -56,27 +56,22 @@ def usage_option_chooser
     session[:usage_options] = @usage_options
```

Comment:
do we need session[:usage_options] & @usage_options, i think references to those should be removed too.